src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const resultObject = {
+    result: 0,
+  };
+
+  resultObject.add = function (a, b) {
+    return a + b;
+  };
+
+  resultObject.subtract = function (a, b) {
+    return a - b;
+  };
+
+  resultObject.multiply = function (a, b) {
+    return a * b;
+  };
+
+  resultObject.divide = function (a, b) {
+    if (b === 0) {
+      return undefined;
+    } else {
+      return a / b;
+    }
```

Comment:
Use the shorthand method syntax for defining `divide` instead of the `function` keyword.